src/client/Microsoft.Identity.Client/Internal/Logger/MsalCacheLoggerWrapper.cs

```diff
@@ -0,0 +1,40 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics.Tracing;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Identity.Client.Core;
+using Microsoft.IdentityModel.Abstractions;
+
+namespace Microsoft.Identity.Client.Internal.Logger
+{
+    internal class MsalCacheLoggerWrapper : IIdentityLogger
```

Comment:
This is the class to be exposed in the TokenCacheNotificationArgs for logging. Im assuming there is no Pii from the logs but I am thinking about adding an extension method to the interface to handle Pii

Reply:
@bgavrilMS